Apps/WebClient/src/ClientApp/src/components/communication.vue

```diff
@@ -18,12 +26,20 @@ import { ICommunicationService } from "@/services/interfaces";
 
 @Component
 export default class CommunicationComponent extends Vue {
-  private communication: Communication | null = null;
+  private communication?: Communication = null;
```

Comment:
nit: not necessary to make it equal to null since the ? implies that is undefined